src/data/faq.json

```diff
@@ -54,6 +54,14 @@
                 "title": "Common Issues",
                 "id": "most_asked",
                 "accordion": [
+                    {
+                        "title": "[Apple/iOS]: Why do I see a screen which tells me to update my operating system?",
+                        "anchor": "update_iOS",
+                        "active": true,
+                        "textblock": [
+                            "This screen is shown on iPhones running an iOS version between iOS 13.5 and (including) iOS 13.6.1. On these iOS Versions, the necessary version 2 of the Exposure Notification Framework is not available, therefore it's necessary to update to iOS 13.7 or newer, to continue to use the Corona-Warn-App. More information can be found in <a href='../faq/#ios135' target='_blank' rel='noopener noreferrer'>this</a> FAQ entry."

```

Comment:
I think that `to continue to use` sounds a bit strange, any ideas for another wording here?

Reply:
"to continue to use" sounds fine to my native ears, but I just noticed that there's now a double use of "necessary" in the sentence, which is just a style point.

You could change "it's necessary to" => "you need to" since the FAQ question is written in the direct "I" form, you can reply also directly using "you".

Reply:
Thanks Mike, I've changed it. And thanks for the confirmation that "to continue to use" is ok.